### PR TITLE
chore(deps): cache cilium 1.13.10

### PR DESCRIPTION
**Pull request by behzad-mir**
**What type of PR is this?**

**What this PR does / why we need it**:
Adding cilium 1.13.10 image to the VHD
**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [X] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -206,7 +207,9 @@
         "1.12.10-2",
         "1.12.10-3",
         "1.13.5-1",
-        "1.13.5-3"
+        "1.13.5-3",
+        "1.13.10",
```
Comment (wedaly) [inline vhdbuilder/packer/components.json:211]: cilium-agent just needs 1.13.10-1, so I think we can remove 1.13.10

Comment (behzad-mir) [reply vhdbuilder/packer/components.json]: addressed in the latest commit

**Review by wedaly — APPROVED** [review]

**Review by camrynl — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -386,7 +388,7 @@
       "downloadLocation": "/opt/crictl/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/cri-tools/v*/binaries",
       "versions": [
-        "1.28.0"
+        "1.29.0"
```
Comment (AbelHu) [inline vhdbuilder/packer/components.json:391]: Is this expected?

**Review by AbelHu — APPROVED** [review]
LGTM. Thanks for the PR. Please assure that the tests pass.

**Pull request merged**